.spelling

```diff
@@ -1190,3 +1190,8 @@ tests.zip
 v5.0
 v6.0.
 #endregion
+
+#region test/tools/WebListener/README.md Overrides
+ - test/tools/WebListener/README.md
+ResponseHeaders
+#endregion
```

Comment:
Are they really all used?

Reply:
I'm not sure what you are asking. But, I added a file level override because I had been putting all of the overrides for this in the Global dictionary.  That was probably a poor decision, so I'm adding the new section and will do a separate PR after this is merged to move the ones from my previous PRs from global to the file override.

Reply:
Thanks. Clear.

Closed.